### PR TITLE
Add Tigris environment

### PR DESCRIPTION
**Pull request by AlekSi**
Add Docker Compose configuration.
Update deps.
Reformat files.

# Checklist

* [ ] Tests are added for new functionality or fixed bugs.
* [x] `task all` passes.

See CONTRIBUTING.md for more details.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -19,6 +19,46 @@ services:
...
+    depends_on: [tigris_fdb]
+    restart: on-failure
+    command: bash -c 'fdbcli --exec "configure new single memory" && /server/service'
+    environment:
+      - TIGRIS_SERVER_SEARCH_AUTH_KEY=ts_test_key
+      - TIGRIS_SERVER_SEARCH_HOST=tigris_typesense
+    ports:
+      - 8081:8081
+    volumes:
+      - tigrisdata:/etc/foundationdb
```
Comment (rumyantseva) [inline docker-compose.yml:60]: This part is unclear to me. There are mentions of `foundationdb` commented in some files, but it's uncommented here. Why?

**Review by rumyantseva — COMMENTED** [review]
Looks ok'ish to me :) 

As there are changes in Dockerfiles, does this PR need the `trust` label? So docker builds are built etc.

**Comment by rumyantseva** [comment]
^ I don't know what I pressed but `help_wanted` was added by mistake indeed.

File: build/deps/tigris_fdb.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,3 @@
+FROM foundationdb/foundationdb:6.3.23
+
+# FROM tigrisdata/foundationdb:7.1.7
```
Comment (w84thesun) [inline build/deps/tigris_fdb.Dockerfile:3]: This and `tigris_typesense` would be uncommented in the next PR, I guess?

Comment (AlekSi) [reply build/deps/tigris_fdb.Dockerfile:3]: When we upgrade to the next Tigris alpha. Currently, I could not make it work

File: docker-compose.yml (hunk 1 above)
Comment (AlekSi) [reply docker-compose.yml:60]: The next version of Tigris will change this path

**Review by w84thesun — APPROVED** [review]

**Review by rumyantseva — APPROVED** [review]
LGTM

**Pull request merged**